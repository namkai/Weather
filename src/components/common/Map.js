import PropTypes from 'prop-types';
import React from 'react';
import { Gmaps } from 'react-gmaps';

const Map = ({ lat, lon }) => (
  <div className="col-md-6">
    <Gmaps
      width={'100%'}
      height={'350px'}
      lat={lat}
      lng={lon}
      zoom={12}
    >
    </Gmaps>
  </div>
);

Map.defaultProps = {
	lat: 1,
	lon: 2,
};

Map.propTypes = {
	lat: PropTypes.number,
	lon: PropTypes.number,
};
export default Map;
