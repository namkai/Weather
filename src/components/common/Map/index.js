import Map from './view';

export default Map;

