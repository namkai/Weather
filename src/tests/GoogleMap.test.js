import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import Map from '../components/Map';

it('renders correctly', () => {
	const map = shallow(<Map />);
	expect(toJson(map)).toMatchSnapshot();
});