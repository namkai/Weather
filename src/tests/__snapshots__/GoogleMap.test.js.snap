// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="col-md-6"
>
  <Component
    height="350px"
    width="100%"
    zoom={12}
  >
    <Component />
  </Component>
</div>
`;
